web/src/helpers/downloadBoleto.js

```diff
@@ -0,0 +1,11 @@
+import { sendLogEvents } from '@/services/logEvent';
+import downloadFile from '@/helpers/downloadFile';
+
+export default async function boletoDownload(url, title = '', type = '') {
```

Comment:
```suggestion
export default async function downloadPayable(url, title = '', type = '') {
```

mudar tudo pra downloadPayable!